### DISCUSSION TITLE
개인 제페토 캐릭터를 유니티에서 따로 포즈를 잡고 표정을 바꾸고 싶어요

### DISCUSSION THREAD

**Discussion by kdrt7878** [naverz/zepeto-studio-kor#1187] [5. Q&A]
안녕하세요, 제페토 내 캐릭터 불러오기까진 성공하였으나 리깅은 되어있으나 해당 리깅된 부분을 잡았을때 팔이나 다리가 움직이지 않고 prefab으로 저장해도 제페토 모델이 제대로 저장되지 않아 불가능합니다 ㅠㅠ 제가 따로 포즈를 잡는 것은 어떻게 해야할지 모르겠습니다 ㅠㅠ

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 월드 내에서는 아쉽지만 원하는 포즈 및 표정 세팅이 불가합니다.
제스쳐 기능 API가 조만간 공개될 예정이오니 그때 어느정도 원하시는 기능을 사용하실 수 있으리라 기대합니다.
감사합니다.
